.github/EnsureHttps.ps1

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env pwsh
+
+Set-StrictMode -Version Latest
+$ErrorActionPreference = 'Stop'
+
+$changedFiles = git diff --name-only "origin/$env:GITHUB_BASE_REF" HEAD -- manifests
+
+foreach ($changedFile in $changedFiles) {
+  $http = Select-String -Path $changedFile `
+    -Pattern '(http://.*)$' `
+    -AllMatches
+  foreach ($match in $http.Matches) {
+    if ($match.Groups[0].Value) {
+      $https = $match.Groups[0].Value.Replace('http://', 'https://')
+      $res = Invoke-WebRequest -Uri $https `
+        -Method HEAD `
+        -TimeoutSec 10 `
+        -UseBasicParsing
+      if ($res.StatusCode -eq 200) {
+        Write-Output "::error file=$changedFile::Found HTTP link: $($match.Value)"
```

Comment:
```suggestion
        Write-Output "::error file=$changedFile::Found HTTPS link: $($match.Value)"
```

Reply:
I'm not sure this makes sense as an error message.

If we are trying to prompt users to upgrade their links from HTTP to HTTPS, an error message that says `Found HTTPS link http://example.com` would be confusing.

Reply:
I had misread, thanks for catching.